app/main.py

```diff
@@ -1 +1,30 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import (
+    NotWearingMaskError,
+    NotVaccinatedError,
+    OutdatedVaccineError
+)
+
+
+def go_to_cafe(
+        friends: list,
```

Comment:
In future tasks, you could be more concrete in type annotations defining that you expect to obtain a list of dicts for `friends`.